infra/terraform/modules/temporal/task_definitions.tf

```diff
@@ -0,0 +1,186 @@
+          { name = "SKIP_DB_CREATE", value = "true" },
+          { name = "SKIP_VISIBILITY_DB_CREATE", value = "false" },
+        ]
+      )
+
+      secrets = local.shared_secrets
+
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          "awslogs-create-group"  = "true"
+          "awslogs-group"         = local.log_group_name
+          "awslogs-stream-prefix" = local.migrate_service_name
+          "awslogs-region"        = var.region
+        }
+      }
+    },
+    {
+      essential = true
+      name      = "${local.prefix}${local.temporal_service_name}"
+      image     = "temporalio/server:${local.temporal_version}"
+      cpu       = 0 # let ECS divvy up the available CPU
+      dependsOn = [{ condition = "SUCCESS", containerName = "${local.prefix}${local.migrate_service_name}" }]
+      healthCheck = {
+        # If we just want to for when the socket accepts connections, we can use this:
+        # command     = ["CMD", "/bin/sh", "-c", "nc -z $(hostname) 7233"]
+
+        # This checks that the server is up and running
+        command     = ["CMD", "/bin/sh", "-c", "temporal operator cluster health --address $(hostname):7233 | grep -q SERVING"]
+        startPeriod = 10
+        interval    = 5
+        retries     = 10
+        timeout     = 5
+      }
+
+      environment = concat(local.shared_env_vars,
+        # container specific env vars go here
+        [
+        ]
+      )
+
+      secrets = local.shared_secrets
+
+      portMappings = [
+        {
+          appProtocol   = "grpc"
+          containerPort = local.temporal_port
+          hostPort      = local.temporal_port
+          protocol      = "tcp"
+        },
+      ]
+
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          "awslogs-create-group"  = "true"
+          "awslogs-group"         = local.log_group_name
+          "awslogs-stream-prefix" = local.temporal_service_name
+          "awslogs-region"        = var.region
+        }
+      }
+    },
+    {
+      readonlyRootFilesystem = true
+
+      essential = false
+      name      = "${local.prefix}${local.setup_service_name}"
+      image     = "${module.setup.url}:${local.temporal_version}"
+      cpu       = 0 # let ECS divvy up the available CPU
+      dependsOn = [
+        { condition = "START", containerName = "${local.prefix}${local.temporal_service_name}" },
+      ]
+
+      environment = concat(local.shared_env_vars,
+        # container specific env vars go here
+        [
+        ]
+      )
+
+      secrets = local.shared_secrets
+
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          "awslogs-create-group"  = "true"
+          "awslogs-group"         = local.log_group_name
+          "awslogs-stream-prefix" = local.setup_service_name
+          "awslogs-region"        = var.region
+        }
+      }
+    },
+  ]
+
+  shared_env_vars = [
+    { name = "DB", value = "postgres12" },
```

Comment:
How is this used? If 12 is the version this doesn't match the version we are using in the local image defined in these files, or remotely (both 15)

Reply:
Understood, thank you